### ISSUE TITLE
[bug] Static anonymous functions reported as breaking PSR2

### ISSUE THREAD

**Issue by kubawerlos** [squizlabs/PHP_CodeSniffer#1775]
When calling `bin/phpcs --standard=PSR2` on

```php
        array_map(
            function ($x) {
                return [$x];
            },
            [1, 2, 3]
        );
```

it is - as expected - perfectly fine, but when adding [static](http://php.net/manual/en/functions.anonymous.php#functions.anonymous-functions.static) to the anonymous functions, like this:
```php
        array_map(
            static function ($x) {
                return [$x];
            },
            [1, 2, 3]
        );
```

it reports:
```bash
------------------------------------------------------------------------------
FOUND 2 ERRORS AFFECTING 2 LINES
------------------------------------------------------------------------------
 2 | ERROR | [x] Line indented incorrectly; expected 8 spaces, found 12
 4 | ERROR | [x] Line indented incorrectly; expected 8 spaces, found 12
------------------------------------------------------------------------------
PHPCBF CAN FIX THE 2 MARKED SNIFF VIOLATIONS AUTOMATICALLY
------------------------------------------------------------------------------
```

Unfortunately `phpcbf` fails on fixing and leaves the code like this:
```php
        array_map(
        static function ($x) {
            return [$x];
        },
            [1, 2, 3]
        );
```

**Comment by jrfnl** [comment]
Duplicate of #1761 ?

**Comment by kubawerlos** [comment]
Yes, it is.
